android/app/build.gradle

```diff
@@ -89,7 +89,7 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
-        versionName "6.1.0"
+        versionName "1.0.0"
```

Comment:
I would start projects on 0.1.0

Reply:
Just to be clear, `1.0.0` will be the _template_ version, and not the _project_ version. The project version is set to `0.0.1` by the CLI script [here](https://github.com/rootstrap/react-native-template/blob/master/cli/setup-project.js#L43).

Reply:
Oh, good to know. Thanks